src/ngChosen.ts

```diff
@@ -79,7 +80,15 @@ module ngChosen {
 			} else {
 				this.updateState(elem, false, false, false);
 			}
-
+            
+            if (scope.ngModel !== undefined) {
+                scope.$watch("ngModel", function(newValue, oldValue) {
```

Comment:
I will add checking on newval and oldvalue.
Did a formatting on my visual studio code. I will revert that too.